components/actiongroup/stories/template.js

```diff
@@ -65,102 +65,69 @@ export const OverflowOption = (context) => Container({
 	},
 	content: html`
 		${Container({
-				heading: "Wrap",
-				content: Template({
-					customStyles: { "max-inline-size": "288px" },
-					content: [
-						{
-							iconName: "Edit",
-							iconSet: "workflow",
-							label: "Edit",
-						},
-						{
-							iconName: "Copy",
-							iconSet: "workflow",
-							label: "Copy",
-						},
-						{
-							iconName: "Delete",
-							iconSet: "workflow",
-							label: "Delete",
-						},
-						{
-							iconName: "Cut",
-							iconSet: "workflow",
-							label: "Cut",
-						},
-						{
-							iconName: "Move",
-							iconSet: "workflow",
-							label: "Move",
-						},
-					]
-				})
-			}
-		)}
+			withBorder: false,
+			heading: "Wrap",
+			content: Template({
+				customStyles: { "max-inline-size": "288px" },
+				content: [
+					{
+						iconName: "Edit",
+						iconSet: "workflow",
+						label: "Edit",
+					},
+					{
+						iconName: "Copy",
+						iconSet: "workflow",
+						label: "Copy",
+					},
+					{
+						iconName: "Delete",
+						iconSet: "workflow",
+						label: "Delete",
+					},
+					{
+						iconName: "Cut",
+						iconSet: "workflow",
+						label: "Cut",
+					},
+					{
+						iconName: "Move",
+						iconSet: "workflow",
+						label: "Move",
+					},
+				]
+			}, context)
+		})}
 		${Container({
-				heading: "Collapse",
-				content: Template({
-					content: [
-						{
-							iconName: "Edit",
-							iconSet: "workflow",
-							label: "Edit",
-						},
-						{
-							iconName: "Copy",
-							iconSet: "workflow",
-							label: "Copy",
-						},
-						{
-							iconName: "Delete",
-							iconSet: "workflow",
-							label: "Delete",
-						},
-						{
-							iconName: "More",
-							iconSet: "workflow",
-							hideLabel: true,
-						},
-					]
-				})
-			}
-		)}
+			withBorder: false,
+			heading: "Collapse",
+			content: Template({
+				content: [
+					{
+						iconName: "Edit",
+						iconSet: "workflow",
+						label: "Edit",
+					},
+					{
+						iconName: "Copy",
+						iconSet: "workflow",
+						label: "Copy",
+					},
+					{
+						iconName: "Delete",
+						iconSet: "workflow",
+						label: "Delete",
+					},
+					{
+						iconName: "More",
```

Comment:
In my screenreader testing, this button was just announced as "toggle button." Do we have a specific label for a button like this, that probably triggers a popover/menu?